services/graphql/src/graphql/definitions/lineitem.js

```diff
@@ -50,6 +51,7 @@ type LineItem implements Timestampable & UserAttributable @applyInterfaceFields
   dates: LineItemDates!
   priority: Int!
   ads(input: LineItemAdsInput = {}): AdConnection! @refMany(modelName: "ad", localField: "_id", foreignField: "lineitemId")
+  externalLinks: [ExternalLink]
```

Comment:
Can this changed to `[ExternalLink!]!` and then ensure that the `externalLinks` field always resolves to an array?